Tests/ActionTests/ActionTests.swift

```diff
@@ -91,6 +105,10 @@ class ActionTests: QuickSpec {
                             next(20, false),
                         ])
                     }
+
+                    it("executes twice") {
+                        XCTAssertEqual(executionObservables.events.count, 2)
```

Comment:
Since the tests use Quick/Nimble, I think using a Nimble matcher instead of `XCTAssert`s might make sense. Something like `expect(executionObservables) == 2`. What do you think?